### ISSUE TITLE
Using a different database name than theme name

### ISSUE THREAD

**Issue by vjmurphy** [Varying-Vagrant-Vagrants/VVV#1690]
I'm having trouble doing something I'm sure I was able to do in VVV 2.0, but my brain has curdled enough that I can't recreate it. 

I'm on the latest version of VVV, with the latest version of Virtual Box, on MAc OS Sierra, 10.12.6. 

I want to create a theme folder with a name different from the DB name. Foe example, murphy for the theme name, and wp_murphy for the database name. 

I've tried using the custom db_name, but it didn't seem to work (or I might have had a syntax problem), and I've also tried an empty WP installation, then importing the backup sql, but no dice. 

I know I had it working previously, but I just can't get it working this time. Are there any clear instructions on how to do this?

**Comment by tomjn** [comment]
The theme name has nothing to do with the database name, did you mean the site name? Can you include what you tried to use in a code block rather than describing with words?

**Comment by vjmurphy** [comment]
It seemed to me that the default config values use the theme name/site name for the database name if nothing is altered. 

I tried adding this to the vvv-custom.yml file: 

interactions:
  description: "Murphy"
  repo: https://github.com/Varying-Vagrant-Vagrants/custom-site-template.git
  hosts:
   - murphy.test
  custom: 
   - db_name: wp_murphy

WIth no success. I've also tried renaming the database in PHPAdmin, also with no success. I'm trying to sync my db name with the production version db name, so that I can sync them up.

**Comment by tomjn** [comment]
Oh, did you make the change **after** you provisioned the site?

For these kinds of changes you need to reprovision, but if you've already provisioned, then `wp-config.php` etc will be set up to use the first database name, not the new one. You'll need to reprovision, then manually adjust `wp-config.php`, or, delete the site from `vvv-custom.yml`, and the folder, and reprovision to recreate it with the new `wp-config.php`. VVV provisioners won't edit existing files once they're in place as that's opening a massive can of worms

All in all, though, some if else statements in `wp-config.php` and some SQL import/export adjustments should mean you don't need the database name to match production for it to work with the same codebase.

**Comment by tomjn** [comment]
Also keep in mind that a `vagrant up` is just an on switch, it won't reprovision/redo the `vvv-custom.yml` unless you `vagrant up --provision` or `vagrant provision`

**Comment by vjmurphy** [comment]
Yep, been doing the provisioning correctly. Let me give your steps a try. Thanks!
